projects/elife.yaml

```diff
@@ -773,6 +774,8 @@ elife-bot:
                     deletion-policy: delete
                 "{instance}-elife-bot-digests-input":
                     deletion-policy: delete
+                "{instance}-elife-bot-decision-letter-input":
```

Comment:
I think we should remove the `s1604` section now that `elife-bot` runs on 16.04 by default; or transform it into `s1804`. @lsh-0?

Reply:
Probably remove it for now, I'll re-introduce it on a per-project basis for the ticket to upgrade all 16.04 projects to 18.04.